style/global.css

```diff
@@ -283,7 +283,7 @@ a {
 }
 .tab {
   display: flex;
-  width: 120px;
+  max-width: 150px;
```

Comment:
Just enough to handle the "Release Notes" tab, and saves a bit of space where tabs aren't using the full width.